src/pages/Home.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './Home.scss';
+
+const Home = () => (
+  <div className="home-container">
+    <h2>Welcome to our page!</h2>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam elementum porta ex sit amet
+      sollicitudin. Sed finibus ornare mi, sit amet vestibulum ligula tempor eget. Praesent quis
+      enim in justo finibus faucibus. Duis pellentesque velit sed massa pellentesque, et posuere
+      lorem placerat. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere
+      cubilia curae; Curabitur eu lorem orci. Donec laoreet viverra quam iaculis eleifend.
+      Aenean sit amet consequat tellus, ut dignissim sem. Aenean placerat tincidunt erat, a luctus
+      sem accumsan non. In ut sapien erat.
+    </p>
+    <p>
+      Aliquam ornare eget velit id pharetra. Fusce efficitur aliquam ullamcorper. Donec
+      tristique, arcu vitae facilisis semper, tellus sem consequat augue, vel dictum nisl
+      nulla sed diam. Donec pellentesque orci a velit eleifend, suscipit laoreet nibh pretium.
+      Praesent ultricies sodales ligula quis sagittis. Vivamus vehicula molestie molestie. Sed
+      tincidunt eget neque sit amet consequat. Aenean turpis enim, gravida a euismod et,
+      feugiat at ante.
+    </p>
+  </div>
```

Comment:
- [ ] [OPTIONAL] Great job done @Alexr16 , however, kindly refer to my recommendation concerning the calculator and implement the same here. Once again, great job 👍🏽